src/main/menu/templates/theme.js

```diff
@@ -37,6 +37,14 @@ export default function (userPreference) {
       click (menuItem, browserWindow) {
         actions.selectTheme('material-dark')
       }
+    }, {
+      label: 'Vibrancy Dark',
+      type: 'radio',
+      id: 'vibrancy-dark',
+      checked: theme === 'vibrancy-dark',
+      click (menuItem, browserWindow) {
+        actions.selectTheme('vibrancy-dark')
+      }
```

Comment:
@ZhuZhengyi The theme should only be available on macOS because on Linux and Windows we cannot set vibrancy. Meaning it's just One-Dark.

Reply:
Yes, it's based on one-dark.